src/main/resources/META-INF/plugin.xml

```diff
@@ -20,7 +20,7 @@
     <vendor>DemonWav, gabizou, and kashike</vendor>
 
     <description><![CDATA[
-      Brings support for Bukkit, Spigot, Paper, BungeeCord, Sponge, LiteLoader, and Forge Projects to Intellij IDEA.
+      Brings support for Bukkit, Spigot, Paper, BungeeCord, Sponge, LiteLoader, Forge and Canary Projects to Intellij IDEA.
```

Comment:
Oxford comma, please.
![x](http://cdn.thewritepractice.com/wp-content/uploads/2011/08/the-oxford-comma.jpg)